nntrainer/src/databuffer_func.cpp

```diff
@@ -135,8 +135,8 @@ int DataBufferFromCallback::setFunc(
   return status;
 }
 
-void DataBufferFromCallback::updateData(BufferType type, int &status) {
-  status = ML_ERROR_NONE;
+void DataBufferFromCallback::updateData(BufferType type) {
+  int status = ML_ERROR_NONE;
```

Comment:
should we throw when `status != ML_ERROR_NONE` at line 199? I can't comment at line 199

Reply:
The data buffer is going to refactored heavily with #305. It would be covered in it.

Reply:
Alright